### ISSUE TITLE
[Firefox] Preferences sync between Android and Desktop

### ISSUE THREAD

**Issue by desbma** [gorhill/uBlock#1244]
Preferences sync between Firefox on Android and Desktop Firefox does not work, as mentioned [in the wiki](https://github.com/gorhill/uBlock/wiki/Cloud-storage):
> It appears Firefox for Android can't sync extensions settings (correct me if I am wrong).

Sync between desktop instances works fine.

uBlock 1.5.5
Firefox 43

**Comment by gorhill** [comment]
> Sync between desktop instances works fine

What extension did you test to confirm this?

**Comment by desbma** [comment]
uBlock (1.5.5 and several previous versions since you added sync support)

**Comment by gorhill** [comment]
No -- you open an issue saying "Sync between desktop instances works fine". How did you conclude that syncing extension settings across Firefox for Android and Firefox works? What extension did you **succeed** syncing its settings?

**Comment by desbma** [comment]
Sorry, I don't understand you.

Let me rephrase:
Let's say I have 2 desktop machines (PC1 and PC2) and an Android Phone, all 3 with Firefox and uBlock.
* Sync of uBlock prefs between PC1 and PC2 **works**
* Sync of uBlock prefs between Phone and PC1 or Phone and PC2 **does not work**

**Comment by gorhill** [comment]
Ok sorry I misunderstood.

> Sync of uBlock prefs between Phone and PC1 or Phone and PC2 **does not work**.

Isn't it what I say in the doc? There is no entry in Firefox for Android to sync extensions settings, unlike on Firefox desktop. The syncing of extension settings is **not** supported by Firefox for Android.

**Comment by gorhill** [comment]
To be clear, I have no idea why the issue has been opened here: the documentation you quote says exactly that Firefox for Android does not support syncing add-on settings, and yet you opened an issue about this. What am I missing?

**Comment by desbma** [comment]
My fault, I should have explained more in the first message :)

The doc is correct, this issue is not about the doc.

> There is no entry in Firefox for Android to sync extensions settings, unlike on Firefox desktop. The syncing of extension settings is not supported by Firefox for Android.

Do you mean this is Firefox limitation? I that case you can close this, I though that it was possible, but just not implemented in uBlock yet.

**Comment by gorhill** [comment]
> Do you mean this is a Firefox limitation?

A Firefox for Android limitation -- extension settings can't be synced -- as stated in the doc.
